src/codegen/spirv/spec.zig

```diff
@@ -1302,7 +1302,7 @@ pub const Decoration = enum(u32) {
     RegisterINTEL = 5825,
     MemoryINTEL = 5826,
     NumbanksINTEL = 5827,
-    BankwidthINTEL = 5828,
+    BandwidthINTEL = 5828,
```

Comment:
This feels like an API break

Reply:
Probably intel jargon

Reply:
Yeah, it's bank, it's unfortunate that it isn't `BankWidthINTEL` to match `BankBitsINTEL`:

Number | Decoration
-- | --
5827 | NumbanksINTEL
5828 | BankwidthINTEL
5835 | BankBitsINTEL